app/controllers/admin/cases_controller.rb

```diff
@@ -106,6 +126,7 @@ def create_params(correspondence_type)
     case correspondence_type
     when 'foi' then create_foi_params
     when 'sar' then create_sar_params
+    when 'ico' then create_ico_params
```

Comment:
We don't actually use `create_params` anywhere ... and I just commented it out and the `create_*_params` methods below and was able to create cases just fine. Let's toast them?